### ISSUE TITLE
make doc doesn't compile while make default does.

### ISSUE THREAD

**Issue by Stenodyon** [severus21/ide_cachan#3]
I'm having inconsistencies between the output of ```make doc``` and ```make``` which are making travis builds fail.

More specifically, ```make doc``` fails with
```
File "src/plugins/ocaml/plugin_fe.mli", line 20, characters 54-63:
Error: Unbound type constructor Core.gset
```
(from https://travis-ci.org/severus21/ide_cachan/builds/182100585)

**Comment by Stenodyon** [comment]
After fixing the error in question with fe0fabe2ad7472875e8c83cbee4e23111531efa8 I was faced with an error I do not know how to solve yet.

```
Finished, 0 targets (0 cached) in 00:00:00.
00:00:00 0    (0   ) STARTING                                                                                                                                                                            -------- |ocamlbuild -use-ocamlfind -package lablgtk2 -package oUnit -package compiler-libs.common ide.docdir/index.html
+ ocamlfind ocamlc -c -package compiler-libs.common -package oUnit -package lablgtk2 -I src/core -I src -I tests -I src/gui -I src/plugins -o src/core/gset.cmo src/core/gset.ml
findlib: [WARNING] Interface topdirs.cmi occurs in several directories: /home/kevin/.opam/4.04.0/lib/ocaml, /home/kevin/.opam/4.04.0/lib/ocaml/compiler-libs
+ ocamlfind ocamlc -c -package compiler-libs.common -package oUnit -package lablgtk2 -I src/core -I src -I tests -I src/gui -I src/plugins -o src/core/miscs.cmi src/core/miscs.mli
findlib: [WARNING] Interface topdirs.cmi occurs in several directories: /home/kevin/.opam/4.04.0/lib/ocaml, /home/kevin/.opam/4.04.0/lib/ocaml/compiler-libs
+ ocamlfind ocamlc -c -package compiler-libs.common -package oUnit -package lablgtk2 -I src/core -I src -I tests -I src/gui -I src/plugins -o src/core/miscs.cmo src/core/miscs.ml
findlib: [WARNING] Interface topdirs.cmi occurs in several directories: /home/kevin/.opam/4.04.0/lib/ocaml, /home/kevin/.opam/4.04.0/lib/ocaml/compiler-libs
Solver failed:
  Ocamlbuild knows of no rules that apply to a target named src/plugins/src/structure/Core.mly. This can happen if you ask Ocamlbuild to build a target with the wrong extension (e.g. .opt instead of .native) or if the source files live in directories that have not been specified as include directories.
Compilation unsuccessful after building 11 targets (0 cached) in 00:00:00.
```

**Comment by Stenodyon** [comment]
Fixed the issue, ```ide.odocl``` was asking for documentation for files that did not exist anymore.
Fixed in commit d69067a56dab319f9f7d9781bbd656e73f18b2fd
